src/main/java/com/team766/controllers/CanSparkMaxSmartMotionRotationalPIDController.java

```diff
@@ -0,0 +1,33 @@
+import java.*;
+
+public class CanSparkMaxSmartMotionRotationalPIDController extends ExtendableCanSparkMaxSmartMotionPIDController {
+	
+	private final double degreesToEncoderUnitsRatio;
+	private final double antiGravK;
+
+	public CanSparkMaxSmartMotionRotationalPIDController(final String configName, final double absEncoderOffset, final double absEncoderOffsetForZeroEncoderUnits, final OffsetPoint first, final OffsetPoint second, double degToEncoderUnitsRatio) {
+		degreesToEncoderUnitsRatio = degToEncoderUnitsRatio;
+		super(configName, absEncoderOffset, absEncoderOffsetForZeroEncoderUnits, first, second);
+	}
+
+	public void setAntigravConstant(double k) {
+		antiGravK = k;
+	}
+
+	
+	//Setpoint is in hall encoder units.
+	public void setNewSetpoint(double setPoint){
+		updateSetpointFromSubclass(setPoint);
+	}
```

Comment:
it looks like `setNewSetpoint` is the same in both subclasses. can it be moved to the superclass, or do you have plans for future developments?

Reply:
done